dp/init.c

```diff
@@ -197,14 +205,120 @@ static inline int port_init(uint8_t port, struct rte_mempool *mbuf_pool)
 	return 0;
 }
 
+#ifdef USE_AF_PACKET
+void
+af_config_network_monitor_interface(uint16_t port_id);
+
+void
+init_af_socks()
+{
+	uint16_t port;
+	struct rte_eth_conf port_conf = port_conf_default;
+	const uint16_t rx_rings = 1, tx_rings = 1;  //tx_rings = rte_lcore_count() - 1;
+	int retval;
+	uint16_t q;
+	char dev_name[LINE_MAX];
+		
+	for (port = S1U_PORT_VETH_ID; port < S1U_PORT_VETH_ID + NUM_SPGW_PORTS; port++) {
+
+		int ifidx;
+		char peer_ifname[IF_NAMESIZE];
+		
+		switch (port) {
```

Comment:
Use an array indexed by loop variable instead?

Reply:
Looks better this way.

Reply:
ok